docs/source/models/cause.rst

```diff
@@ -41,29 +41,251 @@ Common data sources for cause models
+|Measure       |Definition                      |Uses                |
++==============+================================+====================+
+|Prevalence    |Proportion of population        |Initialize cause    |
+|              |with a given condition          |model states        |
++--------------+--------------------------------+--------------------+
+|Birth         |Proportion of all live births   |Initialize neonatal |
+|Prevalence    |born with a given condition     |cause model states  |
++--------------+--------------------------------+--------------------+
+|Incidence     |Number of new cases of a given  |Estimate transition |
+|              |condition per person-year       |rates               |
++--------------+--------------------------------+--------------------+
+|Remission     |Number of recovered cases from a|Estimate transition |
+|              |given condition per person-year |rates               |
++--------------+--------------------------------+--------------------+
+|Disability    |INSERT DEFINITION (Yaqi?)       |INSERT USES (Yaqi?) |
+|Weights       |                                |                    |
++--------------+--------------------------------+--------------------+
+|Cause-Specific|INSERT DEFINITION (Yaqi?)       |INSERT USES (Yaqi?) |
+|Mortality     |                                |                    |
++--------------+--------------------------------+--------------------+
+|Excess        |INSERT DEFINITION (Yaqi?)       |INSERT USES (Yaqi?) |
+|Mortality     |                                |                    |
++--------------+--------------------------------+--------------------+
 
 Prevalence
 ++++++++++
 
-Cause-specific mortality
+Prevalence is defined as the **proportion of a given population that possesses a given condition or trait** at a 
+given timepoint.
+
+	For example, the prevalence of obesity in the United States was approximately 40% in 2016.
+
+When a time *frame* (such as 2016, i.e. 1/1/16 to 12/31/16) instead of a *timepoint* (such as 1/1/16) is 
+reported, it is commonly assumed that the reported prevalence represents the prevalence of the *midpoint* of 
+that timeframe (7/1/16 is the midpoint for the time frame of all of 2016). However, this may not always be the 
+case and it should be considered when relevant.
+
+Prevlance data can be used to **initialize cause model states** and represents the **probability that a simulant 
+will begin the simulation in a given state.**
+
+	For example, the probability that a simulant in a model of obesity in the United States beginning in 
+	2016 will begin the simulation as obese is 0.4 or 40%.
+
+Birth Prevalence
+++++++++++++++++
+
+Birth prevalence is defined as the **proportion of live births in a given population that possess a given 
+condition or trait at birth.**
+ 
+	For example, the birth prevalence for cleft lip in the United States in 2006 was 10.63 per 10,000 live 
```

Comment:
Typo here:
In [2]: (10.63 / 10000) * 100                                                                                           
Out[2]: 0.10630000000000002
0.163% -> 0.106%

Reply:
Thanks, good catch! It's fixed now.